### PR TITLE
Release build 1.1.883.1 to Insider channel

### PR DESCRIPTION
**Pull request by DaveTryon**
#### Describe the change
Release build 1.1.833.1 to the Insider channel. Release notes will automatically be applied to the public release once this PR is merged

> Note: After the PR has been created, certain checks will be kicked off. All of these checks must pass before a merge.

### CODE REVIEW

File: Channels/Insider/release_notes.md [hunk 1]
```diff
@@ -1,45 +1,33 @@
-## April 18th, 2019 Insider Release ([v1.1.0837.02](https://github.com/Microsoft/accessibility-insights-windows/releases/tag/v1.1.0837.02))
+## June 4th, 2019 Insider Release ([v1.1.0883.01](https://github.com/Microsoft/accessibility-insights-windows/releases/tag/v1.1.0883.01))
```
Comment (DaveTryon) [inline Channels/Insider/release_notes.md]: The diff for this file isn't particularly useful. It's probably more useful to review the file contents directly: https://github.com/microsoft/accessibility-insights-windows/blob/d110667477c93efa8763ab4fd66f0f942cbf395a/Channels/Insider/release_notes.md

File: Channels/Insider/release_notes.md [hunk 2]
```diff
@@ -1,45 +1,33 @@
...
+#### Axe.Windows
 
-GitHub users can now file issues directly from inside Accessibility Insights for Windows. Bug filing from Azure DevOps is still supported, and you can freely switch between the two.
+The core scanning code from Accessibility Insights for Windows has been moved into a separate GitHub repo (https://github.com/microsoft/axe-windows). Accessibility Insights now gets its core scanning code from that location. This code will be handled separately to make this functionality more broadly available to the accessibility community.
 
-#### New Rules
-
-We've added the following accessibility rule based on community feedback:
-- Generate a warning if items in a listbox have duplicated names [#133](https://github.com/Microsoft/accessibility-insights-windows/issues/133)
+### Telemetry Improvements
```
Comment (DaveTryon) [inline Channels/Insider/release_notes.md]: Should be H4, not H3
```suggestion
#380 ### Telemetry Improvements
```

File: Channels/Insider/release_notes.md [hunk 3]
```diff
@@ -1,45 +1,33 @@
...
+#### Telemetry Improvements
 
-#### Multiple Release Channels
+We identified a few points in the code where we had no telemetry about failures that users might be experiencing. We added these data points to help us make the product better. No additional customer data is exposed.
 
-You can now choose how frequently your client updates:
-- The **Production** channel updates every 2-4 weeks.
-- The **Insider** channel updates every 1-2 weeks.
-- The **Canary** channel updates multiple times per week.
+#### New Rules
```
Comment (Sirisha-Kommushetty) [inline Channels/Insider/release_notes.md]: Remove New Rules?

File: Channels/Insider/release_notes.md (hunk 1 above)
Comment (Sirisha-Kommushetty) [reply Channels/Insider/release_notes.md]: Look good to me.

**Pull request merged**